partiql-tests-data/eval/query/group-by/group-by.ion

```diff
@@ -1832,6 +1776,7 @@
           supplierId_mixed:11
         },
         {
+          supplierId_mixed:null  // missing coerced to null as per 11.1.1
```

Comment:
This was actually a bug in the Kotlin implementation (tracking issue: https://github.com/partiql/partiql-lang-kotlin/issues/1014). Basically depending on the order of the `FROM` source elements, the group by key name was a different value (missing or null).